pkg/localkube/localkube.go

```diff
@@ -44,7 +44,7 @@ func (l *Localkube) SetUp() error {
 
 	commands := []string{
 		"chmod a+x ./minikube-linux-amd64",
-		"sudo ./minikube-linux-amd64 start --bootstrapper localkube --vm-driver=none --extra-config=apiserver.Authorization.Mode=RBAC --feature-gates=CustomResourceSubresources=true --feature-gates=MountPropagation=false",
+		"sudo ./minikube-linux-amd64 start --bootstrapper localkube --vm-driver=none --extra-config=apiserver.Authorization.Mode=RBAC --feature-gates=CustomResourceSubresources=true,MountPropagation=false",
```

Comment:
Defining the future gates flags multiple times does not work. We have to define the list in a comma separated style like in https://github.com/giantswarm/k8scloudconfig.